<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>322</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionImport_CSV"/>
   </widget>
   <widget class="QMenu" name="menuSql">
    <property name="title">
     <string>Sql</string>
    </property>
    <addaction name="actionSql_Commands"/>
    <addaction name="actionAdd_Notes"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionAdd_Row"/>
   </widget>
   <widget class="QMenu" name="menuSelect_Table">
    <property name="title">
     <string>Select Table</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuQueries">
    <property name="title">
     <string>Queries</string>
    </property>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuSql"/>
   <addaction name="menuData"/>
   <addaction name="menuSelect_Table"/>
   <addaction name="menuQueries"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionImport_CSV">
   <property name="text">
    <string>Import CSV</string>
   </property>
  </action>
  <action name="actionSql_Commands">
   <property name="text">
    <string>Sql Commands</string>
   </property>
  </action>
  <action name="actionAdd_Row">
   <property name="text">
    <string>Add Row</string>
   </property>
  </action>
  <action name="actionAdd_Notes">
   <property name="text">
    <string>Add Notes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
